### DISCUSSION TITLE
Outputs

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1619] [General]
Is there a way for the beta and p value outputs to be standardized in terms of what coordinates are showing? Right now, it appears to be at random so when you're comparing betas and p values, the images move a little. Can we have an option for either the maximum value to be shown? or a matched (or paired) beta and p value?

**Comment by hvgazula** [comment]
Could you please elaborate using an example image?

**Comment by krootesmurdy** [comment]
<img width="1440" alt="Screenshot 2023-01-12 at 12 04 26 PM" src="https://user-images.githubusercontent.com/43792259/212132576-7af506f9-c23e-4bf1-b409-bd75dcc4dd54.png">
<img width="1440" alt="Screenshot 2023-01-12 at 12 04 05 PM" src="https://user-images.githubusercontent.com/43792259/212132588-fe2bc579-e0b9-43fd-a32d-f863ad0ef11c.png">

**Reply by krootesmurdy** [reply]
Do you see how the coordinates don't match up?
